src/moneyTracker.js

```diff
@@ -0,0 +1,21 @@
+function Moneytracker(date, credit, debit, balance) {
```

Comment:
Is `Moneytracker` intended to be at a point where it would pass the acceptance test? Or have you not reached that point?